thorlcr/activities/nsplitter/thnsplitterslave.cpp

```diff
@@ -477,7 +469,7 @@ void CSplitterOutput::stop()
 const void *CSplitterOutput::nextRow()
 {
     ActivityTimer t(totalCycles, activity.queryTimeActivities());
-    if (rec == max)
+    if ((rec == max) && (activity.activeOutputCount>1))
```

Comment:
could possibly set max to RCMAX in start() if activity.activeOutputCount==1 to avoid an extra test on each row.

Reply:
yes good point, making that change.